.github/workflows/ruby.yml

```diff
@@ -24,10 +24,10 @@ jobs:
         key: ${{ runner.os }}-gems-202103-${{ hashFiles('**/Gemfile.lock') }}
         restore-keys: |
           ${{ runner.os }}-gems-202103
-    - name: Use Node.js LTS v18
+    - name: Use Node.js LTS v20
       uses: actions/setup-node@v4
       with:
-        node-version: 'lts/hydrogen'
+        node-version: 'lts/iron'
```

Comment:
Can we do `node-version: '20'` here?  Do we need to know the node version codename?

Reply:
I do.